cmd/operator-sdk/printdeps/cmd.go

```diff
@@ -47,10 +50,36 @@ func printDepsFunc(cmd *cobra.Command, args []string) error {
 	if len(args) != 0 {
 		return fmt.Errorf("command %s doesn't accept any arguments", cmd.CommandPath())
 	}
-	if asFile {
-		scaffold.PrintDepsAsFile()
-	} else if err := scaffold.PrintDeps(); err != nil {
+
+	if err := printDeps(asFile); err != nil {
 		return fmt.Errorf("print deps failed: (%v)", err)
 	}
 	return nil
 }
+
+func printDeps(asFile bool) error {
+	mt, err := projutil.GetDepManagerType()
+	if err != nil {
+		return err
+	}
+	isDep := mt == projutil.DepManagerDep
+
+	// Migrated Ansible and Helm projects will be of type OperatorTypeGo but
+	// their deps files will differ from a vanilla Go project.
```

Comment:
Can you explain this a bit, as I can't see a difference between the helm and ansible functions?

Reply:
Their `PrintDepGopkgTOML` functions are the same but they print different `Gopkg.toml` templates.